packages/office-addin-debugging/src/start.ts

```diff
@@ -290,14 +290,6 @@ export async function startDebugging(manifestPath: string, options: StartDebuggi
             await devSettings.ensureLoopbackIsEnabled(name);
         }
 
-        // enable debugging
-        if (isDesktopAppType && isWindowsPlatform) {
-            await devSettings.enableDebugging(manifestInfo.id, enableDebugging, debuggingMethod, openDevTools);
```

Comment:
This should remain:

await devSettings.enableDebugging(manifestInfo.id, enableDebugging, debuggingMethod, openDevTools);

Please just remove the logging portion below

Reply:
@TCourtneyOwen See my comments. We want the console.log() line, but just not the "Debug method: {method}" part.